### ISSUE TITLE
eslint-disable-next-line comments ignored

### ISSUE THREAD

**Issue by mpumford** [angular-eslint/angular-eslint#1041]
After upgrading from Angular 11 to angular 12  and making the corresponding upgrades to use the correct versions of the angular-eslint modules I've discovered that `eslint-disable-next-line` comments are being ignored when used in a template html file. 
template rules:
```JS
    {
      files: ['*.html'],
      extends: ['plugin:@angular-eslint/template/recommended'],
      rules: {
        '@angular-eslint/template/conditional-complexity': ['error', { maxComplexity: 10 }],
        '@angular-eslint/template/eqeqeq': ['error'],
        '@angular-eslint/template/i18n': [
          'warn',
          {
            checkId: true,
            checkText: true,
            checkAttributes: true,
            ignoreAttributes: [
              'points',
              'd',
              'maxHeight',
              'for',
              'openDirection',
              'NumberInput',
              'link',
              'routerLink',
              'httpEquiv',
              'accept',
              'data-position',
              'maxlength',
              'iconClass',
              'contentWidth',
              'min',
              'accessorType',
              'popupTrigger',
            ],
          },
        ],
      },
    },
```

HTML attempting to disable template/18n. This worked with the angular 11 compatible version
```HTML
        <div class="input-cell">
          <!-- eslint-disable-next-line @angular-eslint/template/i18n -->
          <input
            [(ngModel)]="searchText"
            class="ui input"
            type="text"
            i18n-placeholder="@@searchPlaceholderElip"
            placeholder="Search..."
            name="search"
          />
        </div>
```

**Versions**

| package                                   | version |
| ----------------------------------        | ------- |
| `@angular-eslint/eslint-plugin-template`  | `^12.0.0` |
| `@angular-eslint/template-parser`         | `^12.0.0` |
| `@typescript-eslint/parser`               | `4.28.2` |
| `ESLint`                                  | `^7.26.0` |
| `node`                                    | `14.9.1` |

```sh
$ /home/datav/dv-connect/itron/itron-ui/node_modules/.bin/ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 12.2.17
Node: 14.19.1
Package Manager: yarn 1.22.18
OS: linux x64

Angular: 12.2.16
... animations, common, compiler, compiler-cli, core, forms
... language-service, localize, platform-browser
... platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1202.17
@angular-devkit/build-angular   12.2.17
@angular-devkit/core            12.2.17
@angular-devkit/schematics      12.2.17
@angular/cdk                    12.2.13
@angular/cli                    12.2.17
@schematics/angular             12.2.17
rxjs                            6.5.4
typescript                      4.3.5
```

- [ ] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by mpumford** [comment]
```HTML
        <div class="input-cell">
          <!-- eslint-disable @angular-eslint/template/i18n -->
          <input
            [(ngModel)]="searchText"
            class="ui input"
            type="text"
            i18n-placeholder="@@searchPlaceholderElip"
            placeholder="Search..."
            name="search"
          />
          <!-- eslint-enable @angular-eslint/template/i18n -->
        </div>
```
does work

**Comment by JamesHenry** [comment]
As you note yourself, it is not valid to put an HTML comment inside an element, an internally the Angular Compiler is erroring.

![image](https://user-images.githubusercontent.com/900523/202455627-55718791-6c7f-41ce-9334-50e59bc4ecdb.png)

Closing this one as there is nothing that can be done, you have shown the correct approach - ultimately we are always going to have extra limitations when it comes to our HTML vs our TS
